src/nxdoc/nuxeo-add-ons/nuxeo-aspera-connector.md

```diff
@@ -14,24 +14,399 @@ tree_item_index: 1100
+
+When auto-create is toggled/selected to be “on”, the user needs only to start the upload of the content to the transfer.  Once the upload is completed, the system will automatically start the process to create the content in the system; no need to wait around simply to push a button.  **Before adding content** to the transfer, you can edit the **“common metadata”** which would apply to all items in the transfer.  Users can, of course, decide not to include any metadata at all, and simply add content, too.
+
+{{#> callout type='note' }}
+There are some caveats when using auto-create (auto-create is set to “on”).  Because of some asynchronous behavior, the ability to modify the transfer/file metadata or to add/remove content once your initial "add to transfer" has begun, is not allowed.  If you discover that you have erred in the content/information, you will need to let the transfer process complete before addressing within the system; where the content was added (not the transfer).
+{{/callout}}
+
+If a user has content that needs to be “tagged” or “catalogued” with metadata before creation in the system, the “manual” transition option might be a better choice. To use the **manual option**, choose “no” for auto-create or toggle the auto-create to “off” when creating a new transfer (you can only turn off auto-create if there is **no content in the transfer)**.  This will require someone **manually start** the creation process by clicking the **“complete transfer”** button once the upload is finished (the button is only visible once there is no longer an upload in progress)
+When using the manual transition, the user uploads content to the transfer bucket. While in this step, a user can add content to the bucket and add properties that apply to the items. Properties can be added that "apply to all" using the “common metadata” panel; more file-specific properties can be added either to multiple files at once (bulk edit of metadata) and/or individually,  using a single-file action.
+Once the metadata has been edited to fit the user's needs, the transfer can be **completed** and the content is then created in the system.
+Transfers have 3 states: Draft (no content added yet), In Progress (content added) and Completed (the transfer has been completed and the content has been created in the system).
+
+
+#### Using Aspera with Nuxeo
+
+Once in your instance, Nuxeo Aspera Upload can be accessed from two different ways:
+
+- From the **User Settings** menu located on the left side:
```

Comment:
This one goes to the dashboard

Reply:
"it depends".  it goes to the last aspera view you were in.  if I'm in a trnasfer (not on the dashboard) and then i go to the domain and browse, THEN go to the user settings/aspera dashboard, i land in the transfer I was last in (i just validated on master). not sure i that is was you were addressing here :  )

Reply:
Guess it is a bug then and we should address 😅 
If it says "Aspera Dashboard" it should direct the user to the dashboard 🤷🏼‍♂️